### PR TITLE
Display message tree state admin

### PR DESCRIPTION
**Pull request by Rulios**
Hi

Issue that this PR satisfies: #2066 

Changes: 

- Modified the `oasst_api_client.ts` file to add the API fetch of the endpoint `/api/v1/messages/[message_id]/tree/state`
- Reordered the file structure of `/pages/api/admin/messages/[id]/tree.ts` into `/pages/api/admin/messages/[id]/tree/index.ts` & `/pages/api/admin/messages/[id]/tree/state.ts` for a better understanding of the API route. 
- Added a table in the page `/admin/messages/[id]` that shows the message tree state. 

imagen

Hope it helps :)

### CODE REVIEW

File: website/src/pages/api/admin/messages/[id]/tree/state.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { withRole } from "src/lib/auth";
+import { createApiClient } from "src/lib/oasst_client_factory";
+
+export default withRole("admin", async (req, res, token) => {
+  const client = await createApiClient(token);
+  const messageId = req.query.id as string;
+  const treeState = await client.fetch_message_tree_state(messageId);
```
Comment (notmd) [inline website/src/pages/api/admin/messages/[id]/tree/state.ts:7]: Can you move this call to the `/api/admin/messasges/[id]/tree` endpoint and remove this file?

**Review by notmd — APPROVED** [review]

File: website/src/pages/api/admin/messages/[id]/tree/state.ts (hunk 1 above)
Comment (Rulios) [reply website/src/pages/api/admin/messages/[id]/tree/state.ts:7]: Sure

Comment (Rulios) [reply website/src/pages/api/admin/messages/[id]/tree/state.ts]: Done, I committed it on the PR.

**Comment by notmd** [comment]
Thank you!

**Pull request merged**